_episodes/02-raster-plot.md

```diff
@@ -1,29 +1,25 @@
 ---
-layout: post
-title: "Raster 01: Plot Raster Data in R"
-date:   2015-10-28
+title: "Plot Raster Data in R"
```

Comment:
@jsta do you want to add yourself as an author? i think you should and i think we should talk about author vs contributor because many people in this list were contributors but not authors... details but just a note. regardless you need credit for your work!